.github/workflows/merge.yml

```diff
@@ -15,6 +15,11 @@ on:
 permissions:
   contents: read # to fetch code (actions/checkout)
 
+env:
+  REPO_SLUG: "docker/compose"
+  BUILDX_VERSION: "latest"
+  BUILDKIT_IMAGE: "moby/buildkit:latest"
```

Comment:
Don't think this is needed for compose. Just sticking to default ones should be enough.